ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
Explicitly declaring a test db was never necessary for me to run unit tests.  I'm pretty sure Django will add `test_` to the default db name for creating test dbs.  I'm also pretty sure that Django will use sqlite by default for testing.  I could be wrong, but I thought I'd bring it up.